### DISCUSSION TITLE
Docker Compose Hangs When Cloning Repository in C:\Windows

### DISCUSSION THREAD

**Discussion by sponnify** [danny-avila/LibreChat#1495] [Troubleshooting]
### What happened?

When following the installation instructions for LibreChat on Windows, the docker-compose up command hangs indefinitely. The instructions specify to clone the repository in the root of the C drive, which in this case was C:\Windows. However, Docker does not have the necessary permissions to create directories in C:\Windows, resulting in an “Access is denied” error.

The issue was resolved by cloning the repository into C:\ instead of C:\Windows, but this goes against the current installation instructions as it isn't the "root".

### Steps to Reproduce

Open a terminal in the root of the C drive (C:\Windows).

Run the following commands:

git clone https://github.com/danny-avila/LibreChat
cd LibreChat
copy .env.example .env
docker-compose up

Observe that the docker-compose up command hangs and eventually fails with an “Access is denied” error.

### Relevant log output

```shell
C:\Windows\LibreChat>docker-compose up
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-01-06T01:20:16+08:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
[+] Running 0/0
 - Container chat-mongodb      Creating                                                                            0.0s
 - Container chat-meilisearch  Creating                                                                            0.0s
Error response from daemon: mkdir C:\Windows\LibreChat\meili_data_v1.5: Access is denied.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
similar thing happens on linux when there's permission issues. Try running it from a terminal with elevated permissions (run as admin)

> but this goes against the current installation instructions as it isn't the "root"

I would not recommend installing in `C:\Windows\` and not sure where you saw to do this. If anywhere said "root", what is meant is the root of the drive, which works for you

**Reply by sponnify** [reply]
https://docs.librechat.ai/install/installation/windows_install.html#timestamps states "Open a terminal in the root of the C drive and enter these commands:", but yeah I mistook the root of my C: drive as C:\Windows
